sample/Sample.Tizen/Sample.Tizen.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Tizen.NET.Sdk/1.0.9">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>tizen40</TargetFramework>
+    <TargetFramework>net6.0-tizen</TargetFramework>
   </PropertyGroup>
```

Comment:
no need to specifty the MAUI Version here ? (for a consistent develpment environment)

Reply:
Thanks for your suggestion. I've updated project style and removed unnecessary files.